recasepunc.py

```diff
@@ -402,12 +417,11 @@ def label_for_case(token):
     token = re.sub('[^\p{Han}\p{Ll}\p{Lu}]', '', token)
     if token == token.lower():
         return 'LOWER'
-    elif token == token.lower().capitalize():
+    if token == token.lower().capitalize():
```

Comment:
Why get rid of the elif? If one day we choose to return only at the end of the function, the the logic doesn't flow anymore

Reply:
I would remove elif as it is not needed after a return (as PyLint also highlights by raising a "no-else-return").
If one day we choose to return only at the end of the function, actually the logic will no longer flow, but it will be enough to reintroduce elif.